plasmapy/simulation/particletracker.py

```diff
@@ -248,7 +289,20 @@ def plot_time_trajectories(self, plot="xyz"):  # coverage: ignore
 
     def test_kinetic_energy(self):
         r"""Test conservation of kinetic energy."""
-        assert np.allclose(self.kinetic_energy_history,
-                           self.kinetic_energy_history.mean(),
-                           atol=3 * self.kinetic_energy_history.std()), \
-            "Kinetic energy is not conserved!"
+        conservation = np.allclose(self.kinetic_energy_history,
+                                   self.kinetic_energy_history.mean(),
+                                   atol=3 * self.N * self.kinetic_energy_history.std())
```

Comment:
Okay, I'm an idiot, I increased this tolerance by the N factor because I thought this was some issue with running multiple particles.

BUT IT FAILS FOR A NONZERO ELECTRIC FIELD!

Needs reverting.